core/src/main/java/org/apache/metamodel/CompositeDataContext.java

```diff
@@ -122,7 +123,8 @@ private DataContext getDataContext(Table table) {
                         Schema dcSchema = dc.getSchemaByName(schema.getName());
                         if (dcSchema != null) {
                             // second round = try with schema equals method
-                            if (dcSchema.equals(schema)) {
+                            if (dcSchema.equals(schema) || (dcSchema instanceof DefaultTableAliasedSchema
```

Comment:
I would use the interface `WrappingSchema` here.

Reply:
Updated. Thanks for the tip.